locales/en-US/app.l20n

```diff
@@ -104,8 +103,9 @@
 <contribute "Contribute">
 <nowActive "Now Active">
 <userCount title:"Active Users of this Experiment">
-
 <tourOnboardingTitle "{{title}} enabled!">
 <tourStartButton "Take the Tour">
 <tourCancelButton "Skip">
 <tourDoneButton "Done">
+<userCountContainer "There are <span data-l10n-id="userCount" class="bold" data-hook="install-count"></span> people trying <span data-hook="title"></span> right now!">
```

Comment:
A bit less terse, the user counter number comes from the ampersand model, which gets picked up by the data-hook in the span. I could use the {{title}} template tag, but for consistency sake, I'm bringing that in from the model as well.